src/gui/accountmanager.cpp

```diff
@@ -32,6 +32,7 @@ namespace {
 static const char urlC[] = "url";
 static const char authTypeC[] = "authType";
 static const char userC[] = "user";
+static const char displayNameC[] = "displayName";
```

Comment:
why do we not simply use `const auto` for all those constant strings ?
you could very well do an extra commit to change the status quo and use a more modern way to declare them

Reply:
Done, though I suspect I did a little bit more than what you asked